components/runners/ambient-runner/ambient_runner/bridges/claude/mcp.py

```
@@ -261,6 +327,42 @@ def check_mcp_authentication(server_name: str) -> tuple[bool | None, str | None]
+        try:
+            import urllib.request as _urllib_request
+
+            base = os.getenv("BACKEND_API_URL", "").rstrip("/")
+            project = os.getenv("PROJECT_NAME") or os.getenv(
+                "AGENTIC_SESSION_NAMESPACE", ""
+            )
+            session_id = os.getenv("SESSION_ID", "")
+
+            if base and project and session_id:
+                url = f"{base}/projects/{project.strip()}/agentic-sessions/{session_id}/credentials/gerrit"
+                req = _urllib_request.Request(url, method="GET")
```

Ruff (0.15.6)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)